e2e-tests/init-deploy/compare/statefulset_no-proxysql-pxc.yml

```diff
@@ -76,7 +76,7 @@ spec:
           timeoutSeconds: 5
         name: pxc
         ports:
-        - containerPort: 3306
+        - containerPort: 33062
           name: mysql
           protocol: TCP
```

Comment:
I believe we should expose both 3306 and 33062

Reply:
I have fixed it in code but not in tests. All tests will be fixed soon